docs/references/sdk-references.md

```diff
@@ -0,0 +1,13 @@
+---
+id: sdk-references
```

Comment:
Personally, I think we should start with "what is a Temporal SDK" - and make sure that narrative is more complete - which includes linking to the references.